employee-rostering-webapp/src/test/java/org/optaweb/employeerostering/webapp/employee/EmployeeRestServiceIT.java

```diff
@@ -0,0 +1,133 @@
+
+public class EmployeeRestServiceIT extends AbstractEntityRequireTenantRestServiceIT {
+
+    private EmployeeRestService employeeRestService;
+    private SkillRestService skillRestService;
+
+    public EmployeeRestServiceIT() {
+        employeeRestService = serviceClientFactory.createEmployeeRestServiceClient();
+        skillRestService = serviceClientFactory.createSkillRestServiceClient();
+    }
+
+    @Before
+    public void setup() {
+        createTestTenant();
+    }
+
+    @After
+    public void cleanup() {
+        deleteTestTenant();
+    }
+
+    private Skill createSkill(String name) {
+        Skill skill = new Skill(TENANT_ID, name);
+        Skill out = skillRestService.addSkill(TENANT_ID, skill);
+        assertClientResponseOk();
+        return out;
+    }
+
+    @Test
+    public void testDeleteNonExistingEmployee() {
+        final long nonExistingEmployeeId = 123456L;
+        boolean result = employeeRestService.removeEmployee(TENANT_ID, nonExistingEmployeeId);
+        assertThat(result).isFalse();
+        assertClientResponseOk();
+    }
+
+    @Test
+    public void testUpdateNonExistingEmployee() {
+        final long nonExistingEmployeeId = 123456L;
+        Employee nonExistingEmployee = new Employee(TENANT_ID, "Non-existing employee");
+        nonExistingEmployee.setSkillProficiencySet(Collections.emptySet());
+        nonExistingEmployee.setId(nonExistingEmployeeId);
+        Employee updatedEmployee = employeeRestService.updateEmployee(TENANT_ID, nonExistingEmployee);
+
+        assertClientResponseOk();
+        assertThat(updatedEmployee.getName()).isEqualTo(nonExistingEmployee.getName());
+        assertThat(updatedEmployee.getSkillProficiencySet()).isEqualTo(nonExistingEmployee.getSkillProficiencySet());
+        assertThat(updatedEmployee.getId()).isNotNull().isNotEqualTo(nonExistingEmployee.getId());
+    }
+
+    @Test
+    public void testGetOfNonExistingEmployee() {
+        final long nonExistingEmployeeId = 123456L;
+        assertThatExceptionOfType(javax.ws.rs.NotFoundException.class)
+                .isThrownBy(() -> employeeRestService.getEmployee(TENANT_ID, nonExistingEmployeeId));
+        assertClientResponseError(Response.Status.NOT_FOUND);
+    }
+
+    @Test
+    public void testCrudEmployee() {
+        Skill skillA = createSkill("A");
+        Skill skillB = createSkill("B");
+        Set<Skill> testSkillSet = new HashSet<>();
+        testSkillSet.add(skillA);
+        testSkillSet.add(skillB);
+
+        Employee testAddEmployee = new Employee(TENANT_ID, "Test Employee");
+        testAddEmployee.setSkillProficiencySet(testSkillSet);
+        employeeRestService.addEmployee(TENANT_ID, testAddEmployee);
+        assertClientResponseOk();
+
+        List<Employee> employees = employeeRestService.getEmployeeList(TENANT_ID);
+        assertClientResponseOk();
+        assertThat(employees).usingElementComparatorIgnoringFields(IGNORED_FIELDS).containsExactly(testAddEmployee);
```

Comment:
@Christopher-Chianelli The field ignoring is not needed in this case because `Employee`'s `equals()` overrides the one from `AbstractPersistable` and only looks at `name` and `skillProficiencySet` (it already ignores `id` and `version`). Just FYI. I'm not sure if it's better to remove it because it's not needed (and a bit confusing) or if it's better to keep it for consistency with other tests where it is necessary :man_shrugging:.